### PR TITLE
[TACHYON-1376] Pin Command in separate class. 

### PR DESCRIPTION
**Pull request by CruncherBigData**
added two more tests cases for commands getUsedBytes and getCapacityBytes

### CODE REVIEW

File: integration-tests/src/test/java/tachyon/shell/TfsShellTest.java [hunk 1]
```diff
@@ -506,12 +507,14 @@ public void locationNotExistTest() throws IOException {
...
   public void lsrTest() throws IOException, TachyonException {
+    // clear the loginUser
+    Whitebox.setInternalState(LoginUser.class, "sLoginUser", (String) null);
     MasterContext.getConf().set(Constants.SECURITY_GROUP_MAPPING,
         IdentityUserGroupsMapping.class.getName());
 
     FileInfo[] files = new FileInfo[4];
     String testUser = "test_user_lsr";
-    cleanAndLogin(testUser);
+    System.setProperty(Constants.SECURITY_LOGIN_USERNAME, testUser);
```
Comment (yupeng9) [inline integration-tests/src/test/java/tachyon/shell/TfsShellTest.java]: Do you need this changes?

Comment (yupeng9) [inline integration-tests/src/test/java/tachyon/shell/TfsShellTest.java]: I think we can keep the helper function `cleanAndLogin`

File: integration-tests/src/test/java/tachyon/shell/TfsShellTest.java [hunk 2]
```diff
@@ -1316,4 +1312,47 @@ public void setPinTest() throws Exception {
     Assert.assertEquals(100, mTfs.getInfo(fileA).inMemoryPercentage);
     Assert.assertEquals(0, mTfs.getInfo(fileB).inMemoryPercentage);
   }
+
+  @Test
+  public void getUsedBytesTest() throws Exception {
+    String filePathA = "/testFileA";
+    String filePathB = "/testFileB";
+    int fileSize = 5 * 1024 * 1024 ;
```
Comment (yupeng9) [inline integration-tests/src/test/java/tachyon/shell/TfsShellTest.java]: FYI, you can use Constants.MB.

**Comment by yupeng9** [comment]
Thanks for adding the test coverage! I left some comments.

File: integration-tests/src/test/java/tachyon/shell/TfsShellTest.java (hunk 1 above)
Comment (CruncherBigData) [inline integration-tests/src/test/java/tachyon/shell/TfsShellTest.java]: I did not change that part. I will get a new merge and apply my changes.

File: integration-tests/src/test/java/tachyon/shell/TfsShellTest.java (hunk 2 above)
Comment (CruncherBigData) [inline integration-tests/src/test/java/tachyon/shell/TfsShellTest.java]: excellent suggestion, I will use that.

**Comment by CruncherBigData** [comment]
to comply with the new changes to the shell test case , I added a new command in a separate class for the "Pin Command".

**Comment by yupeng9** [comment]
@CruncherBigData it seems the code is different from what the title suggests. Could you update the title?

**Comment by CruncherBigData** [comment]
done. changed the title.

File: integration-tests/src/test/java/tachyon/shell/command/PinCommandTest.java [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+import tachyon.client.TachyonStorageType;
+import tachyon.client.UnderStorageType;
+import tachyon.client.file.TachyonFile;
+
+import tachyon.shell.AbstractTfsShellTest;
+
+import tachyon.TachyonURI;
+
+public class PinCommandTest extends AbstractTfsShellTest {
+  @Test
```
Comment (apc999) [inline integration-tests/src/test/java/tachyon/shell/command/PinCommandTest.java]: could you also javadoc briefly this testcase?
Exceptions are not required to comment for integration tests.

File: integration-tests/src/test/java/tachyon/shell/command/PinCommandTest.java [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+
+import tachyon.Constants;
+
+import tachyon.client.TachyonFSTestUtils;
+import tachyon.client.TachyonStorageType;
+import tachyon.client.UnderStorageType;
+import tachyon.client.file.TachyonFile;
+
+import tachyon.shell.AbstractTfsShellTest;
+
```
Comment (apc999) [inline integration-tests/src/test/java/tachyon/shell/command/PinCommandTest.java]: empty line 27 29 are unnecessary

**Comment by apc999** [comment]
@CruncherBigData thanks for the patch. The test cases look good to me. I left two minor format or javadoc issues. Thanks!

**Comment by CruncherBigData** [comment]
I have not touched the "lsr" command, not sure why it is failing. resubmitting with fresh version

**Comment by yupeng9** [comment]
LGTM. Thank for adding the test. @CruncherBigData going forward, please you retain the git history, i.e. avoid git commit --amend, so that all the comments are kept in history.

**Pull request merged**